src/js/outreach/apps/verify_app.js

```diff
@@ -52,7 +52,9 @@ export default App.extend({
         .then(() => {
           this.showVerifyCodeView();
         })
-        .catch(() => {
+        .catch(response => {
+          if (response.status >= 500) return;
+
           this.showDialogErrorView();
         });
```

Comment:
This request either returns `204` (success), `400`, or a `500` server error.

With this code update, we’re bailing out of the `.catch()` if it’s a `500` error and letting the error app handle it.

But what should we do if it’s a `400` error? I’m not sure what scenario would cause a `400` error to be returned when a user is requesting a new code be created.

This current code will show the same `Uh oh…` view as the `500` error without routing to the `/outreach/500` URL. Which means the `Try Again` link won’t work, since `history.back()` will have nothing to go to.

So I guess we could either route to the `/outreach/500` via a `Radio.trigger('event-router', 'error’)`. Or create a new view in the `/outreach/views/verify_views.js` file that has a different error message.

Reply:
The previous error page said something about reloading the page, but that didn't work for 500s because it would reload the 500 route.  Really we need one view for 500s and the other view for 400s that can be the original message instead of using one solution for both.

Reply:
I added a general error for the verify app. Which is now shown for that `400` error. Or any other errors besides `500` on the `/api/outreach/otp` request.

Outreach coverage is now at 100% 😄